src/platforms/ios/iostunnelmanager.swift

```diff
@@ -47,7 +47,7 @@ class TunnelManager {
         }
     }
     
-    static func withTunnel(_ f: (_ tunnel: NETunnelProviderManager) throws -> Any) -> Any? {
+    @discardableResult static func withTunnel(_ f: (_ tunnel: NETunnelProviderManager) throws -> Any) -> Any? {
```

Comment:
This allows us to ignore the result, which gives some cleaner code where we were using this.